osutil/kcmdline/kcmdline.go

```diff
@@ -473,3 +473,18 @@ func (kap *ArgumentPattern) UnmarshalYAML(unmarshal func(interface{}) error) err
 	}
 	return kap.unmarshalFromString(arg)
 }
+
+// FilterOut parses a kernel command line and remove arguments that
+// matches match patterns.
+func FilterOut(cmdline string, match []ArgumentPattern) []string {
+	parsedDefault := Parse(cmdline)
+	matcher := NewMatcher(match)
+	keepDefaultArgs := []string{}
```

Comment:
This style is preferred for variables that just use their default initializer
```suggestion
	var keepDefaultArgs []string
```